app/controllers/internal/permissions_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Internal::PermissionsController < Internal::ApplicationController
+  layout "internal"
+
+  def index
+    @users = User.with_role(:admin).to_a + User.with_role(:super_admin).to_a + User.with_role(:single_resource_admin, :any)
```

Comment:
I think you can use union here:

`User.with_role(:admin).union(User.with_role(:super_admin)).union(User.with_role(:single_resource_admin, :any))`

:)

Reply:
In the back of my mind I knew somebody would tell me the better way to do this, so I didn't bother looking it up. Next level dev laziness on my part.